vspace/src/main/java/edu/asu/diging/vspace/web/staff/forms/ExhibitionConfigurationForm.java

```diff
@@ -0,0 +1,72 @@
+package edu.asu.diging.vspace.web.staff.forms;
+
+import java.util.List;
+
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+
+import edu.asu.diging.vspace.core.model.ExhibitionModes;
+
+public class ExhibitionConfigurationForm {
+    
+    private String customMessage;
+    
+    private String defaultExhibitLanguage;    
+    
+    private String exhibitionParam;
+    
+    @NotEmpty(message = "Please select exhibition languages")
+    private List<String> exhibitLanguage;
+    
+    @NotNull(message = "Please select exhibit mode")
+    private ExhibitionModes exhibitMode;
+    
+    @NotEmpty
+    private String spaceParam;
```

Comment:
what's space param?

Reply:
it was passed from the form, removed it from here now